integration-tests/README.md

```diff
@@ -194,6 +194,12 @@ Secret - to create secret
 
 Command to run the tests:
 ```
+export DOCKER_USERNAME=<docker_username>
+export DOCKER_PASSWORD=<docker_password>
+export DOCKER_EMAIL=<docker_email>
+or
+make sure the weblogic image i.e. store/oracle/weblogic:12.2.1.3 already exists locally
+
```

Comment:
NODEPORT_HOST below should be K8S_NODEPORT_HOST

Reply:
Yes, will fix it.